vendor/src/github.com/bsphere/le_go/certs.go

```diff
@@ -0,0 +1,222 @@
+package le_go
+
+// Certs from https://logentries.com/doc/certificates
+var pemCerts = []byte(`
```

Comment:
Using hard-coded certs scares me a little.
It would be better to rely on the host's certificate store.
As is, if the logentries cert was ever compromised it would require releasing a new version of Docker to fix.